pkg/motor/registry_test.go

```diff
@@ -11,7 +11,25 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func Test_CreateAccount(t *testing.T) {
+func Test_CreateAccountWithKeyring(t *testing.T) {
+	req := mt.CreateAccountRequest{
+		Password: "password123",
+	}
+
+	m, _ := EmptyMotor(&mt.InitializeRequest{
+		DeviceId: "test_device",
+	}, common.DefaultCallback())
+	_, err := m.CreateAccount(req)
+	assert.NoError(t, err, "wallet generation succeeds")
+
+	b := m.GetBalance()
+	log.Println("balance:", b)
+
+	// Print the address of the wallet
+	log.Println("address:", m.Address)
+}
+
+func Test_CreateAccountWithKeys(t *testing.T) {
```

Comment:
for tests requiring the chain can there be a "skip" field added for not running in ci?